osquery/tables/system/ssh_keys.cpp

```diff
@@ -0,0 +1,59 @@
+#include <string>
+#include <vector>
+
+#include <osquery/core.h>
+#include <osquery/tables.h>
+#include <osquery/filesystem.h>
+
+#include "osquery/core/conversions.h"
```

Comment:
Are you using any methods from these local includes: `conversions.h` and `system_utils.h`?